index.js

```diff
@@ -132,16 +130,17 @@ UTP.prototype._closeMaybe = function () {
   if (this._closing && !this.connections.length && !this._sending.length && this._inited && !this._closed) {
     this._closed = true
     binding.utp_napi_close(this._handle)
+  } else {
+    for (const conn of this.connections) {
```

Comment:
I didn't understand how these lines could be missing: When you close the server, any open connections are supposed to be closed, right?

Reply:
No, each connection must be closed individually

Reply:
(Same as tcp)

Reply:
The instances created in the tests are not properly torn down then, causing a lot of weird errors happen when I tried to remove this. Checking the tests for "why they fail because this is removed" is the reason this takes to long for me to continue.